data/com.endlessm.HackComponents.appdata.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.endlessm.HackComponents.desktop</id>
```

Comment:
Same comment about the sound server, we don't have a desktop file for this module.

Reply:
I commented in the Sound Server one (I think this is the correct way).